main.js

```diff
@@ -2597,7 +2598,11 @@ function checkVersions(id, deps, globalDeps) {
                         continue;
                     }
 
-                    if (!checkVersion(id, dep, deps[dep], instances)) {
+                    if (globalDeps[dep] !== undefined) {
```

Comment:
Maskes that sense?? gobInstances are filtered in next loop outside this time ... or combine it then you need to also join the objects

Reply:
yeah didnt made sense, double checking in some cases..